app/templates/buyers/brief_responses.html

```diff
@@ -0,0 +1,95 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Responses to {{ brief_data.title or brief_data.lotName }} – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+    {%
+        with items = [
+        {
+            "link": "/",
+            "label": "Digital Marketplace"
+        },
+        {
+            "link": "/buyers",
+            "label": "Your Requirements"
+        }
+        ]
+    %}
+        {% include "toolkit/breadcrumb.html" %}
+    {% endwith %}
+{% endblock %}
+
+{% set responses_count = response_counts['failed'] + response_counts['eligible'] %}
+
+{% block main_content %}
+<div class="grid-row">
+    <div class="column-two-thirds">
+        {% with
+        heading = ("Responses to &lsquo;" + brief_data.get('title', brief_data['lotName']) + "&rsquo;")|safe,
+        smaller = true
+        %}
+        {% include "toolkit/page-heading.html" %}
+        {% endwith %}
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <div class="marketplace-paragraph">
+            <p>{{ responses_count }} supplier{% if responses_count != 1 %}s{% endif %} responded to your requirements. 
```

Comment:
This could be more thoroughly handled with Jinja2's i18n support (pluralize) http://jinja.pocoo.org/docs/dev/templates/#i18n